### PR TITLE
Topic recognition OASIS Brain Dataset 44340326

### PR DESCRIPTION
**Pull request by ArunHarish**
Hi, this is Arun Harish Balasubramonian. I worked on sixth task: Create a generative model of the OASIS brain or the OAI AKOA knee data set using a DCGAN. I worked on the OASIS Brain dataset. 

Student Number : 44340326

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Sufficiently detailed commit messages
- Code valid for Hard Difficulty
- Generated images slightly fuzzy but acceptable

Other Feedback
- Overall good. I would divide up the train function into smaller self-contained functions to make it easier to track data flow.

**Comment by rexsw** [comment]
Code looks good for hard difficulty

For next time;

Comments such as 
      # Binary Cross entropy loss function for generator
     def generator_loss_func(self, output):
Add nothing to the readability of the code and should be avoided. Overall comments were well done

File: recognition/OASIS-Brain-DCGAN-44340326-Arun_Harish_Balasubramonian/driver_test_script.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import sys
```
Comment (shakes76) [inline recognition/OASIS-Brain-DCGAN-44340326-Arun_Harish_Balasubramonian/driver_test_script.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
Results not great
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. remove unnecessary variable and model files.**
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.
